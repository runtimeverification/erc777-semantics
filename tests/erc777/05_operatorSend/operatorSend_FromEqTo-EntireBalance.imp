test.preamble();

// Testing operatorSend(From,To,Value, "", "") when From = To.
// Send entire Balance

test.makeCaller(9);           // switch to 9
test.operatorSend(7, 7, 70, "", "");  // send entire balance
