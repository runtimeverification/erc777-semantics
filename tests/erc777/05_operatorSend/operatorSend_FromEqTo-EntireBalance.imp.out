Changing the caller to 9
9 is sending 70 from 7 to 7 (data:""),  (operatorData:"")
Balance of 7 is 70
Balance of 7 is 70
